### ISSUE TITLE
Confusing error message regarding -print

### ISSUE THREAD

**Issue by davidcok** [dafny-lang/dafny#2617]
The -print option puts out a Boogie file. But the option must have a filename (or `-`); otherwise the message `Error: No input files were specified` is emitted. This is confusing because the missing file is an output, not an input. It could also be more specific.

**Comment by cpitclaudel** [comment]
Agreed, but the error message is right (unfortunately).  Specifically, when you write `Dafny /print input.dfy`, this is parsed as `Dafny /print:input.dfy` — so what's missing is actually the input file, not the output file (the output file is `input.dfy`.

**Comment by davidcok** [comment]
Which is even worse then, because the 'input.dfy' will then be overwritten with a bel file.

**Comment by cpitclaudel** [comment]
> will then be overwritten with a bpl file.

(If you give an additional input, yes)

**Comment by davidcok** [comment]
I suggest the following check on the CLI arguments:

if the file named for /print is given as a separate argument (that is not with : ) and does not either have a .bpl extension or is '-', then the dafny command fails with the error "if the argument to /print is not a .bpl file, then it must be stated with the ':' syntax"
